Find a functionally equivalent code:android.app.ActivityManager.dumpPackageStateStatic:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -1,14 +1,16 @@
 {
     FileOutputStream fout = new FileOutputStream(fd);
     PrintWriter pw = new FastPrintWriter(fout);
-    dumpService(pw, fd, Context.ACTIVITY_SERVICE, new String[] { "package", packageName });
+    dumpService(pw, fd, "package", new String[] { packageName });
+    pw.println();
+    dumpService(pw, fd, Context.ACTIVITY_SERVICE, new String[] { "-a", "package", packageName });
+    pw.println();
+    dumpService(pw, fd, "meminfo", new String[] { "--local", "--package", packageName });
     pw.println();
     dumpService(pw, fd, ProcessStats.SERVICE_NAME, new String[] { packageName });
     pw.println();
     dumpService(pw, fd, "usagestats", new String[] { "--packages", packageName });
     pw.println();
-    dumpService(pw, fd, "package", new String[] { packageName });
-    pw.println();
     dumpService(pw, fd, BatteryStats.SERVICE_NAME, new String[] { packageName });
     pw.flush();
 }

